### ISSUE TITLE
Tests failing in master and branch 7.1

### ISSUE THREAD

**Issue by giddyhut** [kawansoft/aceql-http#21]
stAnalyzer.isWithSemicolons(): true
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.145 sec <<< FAILURE!
test(org.kawanfw.test.api.server.StatementAnalyzerTest)  Time elapsed: 0.093 sec  <<< FAILURE!
java.lang.AssertionError: expected:<false> but was:<true>
        at org.junit.Assert.fail(Assert.java:89)

**Comment by ndepomereu** [comment]
Hi,
As explained in the README in src/test/java:

```
AceQL
-----

All majors tests are run using The Client SDK (aceql-http-client-sdk project):

    <dependency>
        <groupId>com.aceql</groupId>
        <artifactId>aceql-http-client-jdbc-driver</artifactId>
        <version>6.1</version>
    </dependency>
```

**Comment by ndepomereu** [comment]
This is fixed in 7.2, I close the issue.
